libs/application/templates/financial-aid/src/lib/utils.ts

```diff
@@ -8,3 +10,15 @@ export const isValidPhone = (value: string) => {
   return Boolean(phone && phone.isValid())
 }
 export const isValidNationalId = (value: string) => kennitala.isValid(value)
+
+export function hasSpouse(context: ApplicationContext) {
+  const {
+    externalData,
+    answers,
+  } = (context.application as unknown) as FAApplication
```

Comment:
as unknown, do we not have type for application?

Reply:
Yes this is bacially doing that, we know that our application is of type `FAAppliction` but application context is {} so we have to cast it first to unknown to be able to cast it to our application :( 

Welcome to the application system ❤️